src/Avalonia.Themes.Fluent/Controls/ProgressBar.xaml

```diff
@@ -21,6 +21,7 @@
     <Setter Property="Template">
       <ControlTemplate>
         <Border x:Name="ProgressBarRoot" ClipToBounds="True" Background="{TemplateBinding Background}" BorderBrush="{TemplateBinding BorderBrush}" BorderThickness="{TemplateBinding BorderThickness}" CornerRadius="{TemplateBinding CornerRadius}">          
+          <Grid>
```

Comment:
If there is no row or columns, better to go with Panel instead. Less overhead.

Reply:
> What happens to the percentage text color when progress is >=50%. Seems like it wouldn't be visible.
> 
> 1. I think  lightweight resource and potentially another brush property are needed to control this.
> 2. A new property should be added to disable this without re-templating.

The Default theme has a similar issue. I will try to make a new pr to fix both of them in the near future.

Reply:
> If there is no row or columns, better to go with Panel instead. Less overhead.

Fixed that on the current commit.